### DISCUSSION TITLE
Add calcardbackup

### DISCUSSION THREAD

**Discussion by aleyoscar** [nextcloud/all-in-one#3696] [Ideas] [answered]
Hello, I would like to access the `/var/www/html` folder somehow, mainly to use [this script](https://codeberg.org/BernieO/calcardbackup) which provides backups of all the calendars and contacts as .ics and .vcf files. The script requires the location of the `/var/www/html` folder but I'm not sure how to provide that since it is inside the docker container? What would be the best way to use this script with the nextcloud-aio container? Or is there another way to automate exports of all my calendars and contacts to .ics and .vcf files?

Some notes:
- I realize I can just use the borg backup (I do have daily backups enabled) but I would like to have copies of all my calendars and contacts as .ics and .vcf files.
- I realize there is an automatic backup option with the android app, but I have all of my data on external storage and there is no way to specify what directory the backups get saved to (it just saves it to the root folder outside of the external storage)

**Comment by szaimen** [comment]
Hi, so the most elegant way would be to add a community container: https://github.com/nextcloud/all-in-one/tree/main/community-containers#how-to-add-containers. Feel free to ping me if you should need help on this!

Regarding the json, I just leave this here as an example:

```json
{
    "aio_services_v1": [
        {
            "container_name": "nextcloud-aio-calcardbackup",
            "display_name": "Cal and card backup",
            "documentation": "https://github.com/nextcloud/all-in-one/tree/main/community-containers/calcardbackup",
            "image": "waja/calcardbackup",
            "image_tag": "latest",
            "restart": "unless-stopped",
            "environment": [
                "INIT_BACKUP=1"
                "BACKUP_DIR=/backup",
                "NC_DIR=/nextcloud",
                "NC_HOST=%NC_DOMAIN%",
                "DB_HOST=nextcloud-aio-database",
                "DB_PORT=5432"
            ],
            "volumes": [
                {
                    "source": "nextcloud_aio_calcardbackup",
                    "destination": "/backup",
                    "writeable": true
                },
                {
                    "source": "nextcloud_aio_nextcloud",
                    "destination": "/nextcloud",
                    "writeable": false
                }
            ],
            "backup_volumes": [
                "nextcloud_aio_calcardbackup"
            ]
        }
    ]
}
```

**Reply by szaimen** [reply]
Hi @oscarale-camorlinga since you are very interested in this feature IIRC, would you be up for creating the PR that adds this?

**Comment by szaimen** [comment, marked as answer]
This was resolved with https://github.com/nextcloud/all-in-one/pull/5864
